src/displayapp/screens/WeatherSymbols.cpp

```diff
@@ -59,3 +59,28 @@ const char* Pinetime::Applications::Screens::Symbols::GetCondition(const Pinetim
       return "";
   }
 }
+
+const char* Pinetime::Applications::Screens::Symbols::GetSimpleCondition(const Pinetime::Controllers::SimpleWeatherService::Icons icon) {
+  switch (icon) {
+    case Pinetime::Controllers::SimpleWeatherService::Icons::Sun:
+      return "Clear";
+    case Pinetime::Controllers::SimpleWeatherService::Icons::CloudsSun:
+      return "Clouds";
+    case Pinetime::Controllers::SimpleWeatherService::Icons::Clouds:
+      return "Clouds";
+    case Pinetime::Controllers::SimpleWeatherService::Icons::BrokenClouds:
+      return "Clouds";
```

Comment:
You can combine these:

```suggestion
    case Pinetime::Controllers::SimpleWeatherService::Icons::CloudsSun:
    case Pinetime::Controllers::SimpleWeatherService::Icons::Clouds:
    case Pinetime::Controllers::SimpleWeatherService::Icons::BrokenClouds:
      return "Clouds";
```